dojo/tools/factory.py

```diff
@@ -108,6 +109,8 @@ def import_parser_factory(file, test, active, verified, scan_type=None):
         parser = BurpXmlParser(file, test)
     elif scan_type == "Burp Enterprise Scan":
         parser = BurpEnterpriseHtmlParser(file, test)
+    elif scan_type == "Burp REST API":
```

Comment:
Suggest to name it "Burp RESP API report" or similar. We also have other imports such as SonarQube API which actually goes out to the sonarqube API to retrieve data.

Reply:
nvm I see that there's already some info on the actual dropdown further up in the PR.

Reply:
Sonarqube parser do the same.
```
SCAN_SONARQUBE_API = 'SonarQube API Import'
```

Reply:
yes, but that scan is actually import findings via the sonarcube api. The burp rest api parser processes a file that is uploaded to defect dojo.

Reply:
ok